src/main/java/ua/okwine/productexpirationdate/service/ProductService.java

```diff
@@ -0,0 +1,30 @@
+package ua.okwine.productexpirationdate.service;
+
+import ua.okwine.productexpirationdate.entity.Product;
+import ua.okwine.productexpirationdate.entity.Provider;
+import ua.okwine.productexpirationdate.requestWrappers.ProductRequest;
+import ua.okwine.productexpirationdate.requestWrappers.ProductRequestWrapper;
+
+import java.util.List;
+
+public interface ProductService {
```

Comment:
If you have only one implementation there is no need in creation of the interface